### PR TITLE
Add get_class_data.py

### PR DESCRIPTION
**Pull request by skearnes**
Add a script to create dataset pickles for classification datasets with separate files for active and inactive molecules (e.g. DUD-E). Uses the more generic "id" and "assay" in place of "cid" and "aid", respectively.

Question: should we use one of {"mol", "mol_id", "mid"} instead of "id" to be more explicit that it refers to a molecule?

### CODE REVIEW

**Comment by skearnes** [comment]
I simplified this script and changed to used `mol_id` and `assay_id` as the column labels (more descriptive and accurate). PTAL.

File: vs_utils/scripts/public_data/get_class_data.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+"""Build data frames for non-PCBA classification datasets."""
```
Comment (rbharath) [inline vs_utils/scripts/public_data/get_class_data.py]: What about something like "for classification datasets with separate files for active and inactive molecules (e.g. DUD-E)" instead? Other non-PCBA datasets could have different format.s

**Comment by rbharath** [comment]
LGTM (after comment fixed).

File: vs_utils/scripts/public_data/get_class_data.py (hunk 1 above)
Comment (skearnes) [inline vs_utils/scripts/public_data/get_class_data.py]: Done.

**Pull request merged**